lib/assessment.rb

```diff
@@ -6,34 +6,53 @@
 # Make all of those attributes read-only.
 
 ##
-# your code here
+class Dog
+  def initialize(name,breed,age)
+    @name,@breed,@age = name,breed,age
+  end
+
+  attr_reader :name,:breed,:age
```

Comment:
Best practice is to put setters/getters at the top of the class before `initialize`.